src/content/configuration/module.mdx

```diff
@@ -49,6 +49,27 @@ module.exports = {
 };
 ```
 
+## `module.unsafeCache`
+
+`boolean` `function()`
+
+Cache the resolving of module requests. There are couple of defaults for `module.unsafeCache`:
```

Comment:
Should it be `the resolution of` vs `resolving of`, to me, i read like resolving is continuous process, and if you say `the resolution of`, would mean it was resolved and given resolution was cached?

Reply:
resolution of sounds better